main/dpkg/APKBUILD

```diff
@@ -23,7 +22,6 @@ prepare() {
 }
 
 build() {
-	cd "$builddir"
```

Comment:
Hmm, isn't this the 'normal' pattern of APKBUILD files? Why would we want to stray from the normal?